guide/sections/part1/data-publisher.adoc

```diff
@@ -1,23 +1,204 @@
 
-==== How to publish data to the Global Broker
+All NCs and DCPCs are affiliated with a Global Information System Centre (GISC) that has a responsibility to help establish efficient and effective data sharing on the WIS. Your GISC will be able to help you in getting your data onto WIS2.
+
+You may be able to identify an existing NC or DCPC that can publish your data. Alternative, you need to establish a new NC or DCPC. The main difference is that an NC is designated by a Member, whereas a DCPC is designated by a WMO or related international programme and/or a regional association.
+
+Both NC and DCPC require operation of a WIS2 Node (see section 2.4.1. WIS2 Node (TODO CROSSREF)). The procedure for registering a new WIS2 Node is provided in Section 2.6.1.1. Registration and decommissioning of a WIS2 Node (TODO CROSSREF). 
+
+Once you have determined the scope of your datasets, the data policy which applies, and have a WIS2 Node ready for data publication, you are ready to progress to the next step: providing discovery metadata.
+
+==== How to provide discovery metadata to WIS2
+
+Discovery metadata is the mechanism by which you tell potential consumers about your data, how it may be accessed, and any conditions you may place on the use of the data.
+
+Each dataset you want to publish must have an associated discovery metadata record. This record is encoded as GeoJSON (RFC 7946footnote:[RFC 7946 - The GeoJSON Format https://www.rfc-editor.org/rfc/rfc7946]) must conform to the WMO Core Metadata Profile 2 (WCMP2) specification (TODO CROSSREF).
+
+Copies of all discovery metadata records from WIS2 are held at the Global Discovery Catalogues, where data consumers can search and browse to find data that is of interest to them. 
+
+Depending on local arrangements, your GISC may be able to help you transfer your discovery metadata record(s) to the Global Discovery Catalogues. If this is not the case, you will need to publish the discovery metadata record(s) yourselffootnote:[In future, WIS2 may provide metadata publication services (e.g., through a WIS2 metadata management portal) to assist with this task. However, such a service is not available at this time.] using one of two ways:
+
+* The simplest method is to encode the discovery metadata record as a file and publish it to an HTTP server where it can be accessed with a URL. 
+* Alternatively, you may operate a local metadata catalogue through which discovery metadata records can be shared using an API (e.g., OGC API - Recordsfootnote:[OGC API - Records - Part 1: Core https://docs.ogc.org/DRAFTS/20-004.html]). Each discovery metadata record can be accessed with a unique URL via the API (e.g., as an item which is part of thediscovery metadata catalogue).
```

Comment:
is part of the discovery metadata catalogue

Reply:
Fixed.